src/popup/components/AccountDetailsBase.vue

```diff
@@ -104,98 +102,89 @@ export default defineComponent({
     OpenTransferReceiveModalButton,
     TransactionAndTokenFilter,
     BtnClose,
+    IonRouterOutlet,
   },
   props: {
     withoutDefaultButtons: Boolean,
   },
   setup() {
+    const headerEl = ref<HTMLDivElement>();
+
     const route = useRoute();
     const store = useStore();
 
     const { activeAccount } = useAccounts({ store });
-    const ACCOUNT_INFO_HEIGHT = 120;
-    const BALANCE_AND_ACTIONS_HEIGHT = 280;
-    const accountDetailsElem = ref<HTMLElement>();
-    const appInnerScrollTop = ref<number>(0);
-    const clientHeight = ref<number>(0);
-    const initialClientHeight = ref<number>(EXTENSION_HEIGHT);
 
     const { resetFilter } = useTransactionAndTokenFilter();
 
+    const { isScrollEnabled } = useScrollConfig();
+
     const { homeRouteName } = useUi();
 
     const { balance } = useBalances({ store });
 
-    const balanceNumeric = computed(() => balance.value.toNumber());
+    const routerHeight = ref<string>();
 
-    const appInnerElem = computed<HTMLElement | null | undefined>(
-      () => accountDetailsElem.value?.parentElement,
-    );
+    const balanceNumeric = computed(() => balance.value.toNumber());
 
     const routeName = computed(() => route.name);
 
-    const showFilters = computed<boolean>(() => (
-      clientHeight.value > initialClientHeight.value
-        && appInnerScrollTop.value >= ACCOUNT_INFO_HEIGHT
-    ));
+    const showFilters = computed<boolean>(() => (isScrollEnabled.value));
 
-    const resizeObserver = new ResizeObserver(debounce((entries) => {
-      if (!(Array.isArray(entries) && entries.length)) {
-        return;
-      }
-
-      const newClientHeight = entries[0].target.clientHeight;
-
-      const totalHeight = clientHeight.value + BALANCE_AND_ACTIONS_HEIGHT + ACCOUNT_INFO_HEIGHT;
+    function calculateRouterHeight() {
+      const ionicWrapperBottom = document.querySelector('.app-wrapper')?.getBoundingClientRect()?.bottom;
```

Comment:
Please avoid matching CSS classes that are not present in the scope of the current component.